const errorGenerator = () => {
  throw new Error("Fake Error Happend");
};

export default errorGenerator;
